src/symbols.rs

```diff
@@ -115,6 +116,29 @@ pub mod bar {
         one_eighth: ONE_EIGHTH,
         empty: " ",
     };
+
+    #[derive(Debug, Clone, Eq, PartialEq, Hash, Default)]
+    pub enum EmptyBarSymbol {
```

Comment:
Perhaps just some pub consts instead of an Enum for these?

Reply:
Now that this has been moved and includes a `Custom(String)` case I think it makes sense to remain as a enum.